spack/packages/dla-future/package.py

```diff
@@ -53,5 +56,8 @@ def cmake_args(self):
 
         # TESTs
         args.append(self.define('DLAF_WITH_TEST', self.run_tests))
+        
+        # MINIAPPS 
+        args.append(self.define('DLAF_BUILD_MINIAPPS', 'miniapps'))
```

Comment:
Shouldn't this be [self.define_from_variant](https://spack.readthedocs.io/en/latest/spack.build_systems.html#spack.build_systems.cmake.CMakePackage.define_from_variant)?

Reply:
Yes, good catch!